TaskWithCharAndWchar_t/TaskWithCharAndWchar_t.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#define COUNTOF(a) (sizeof(a) / sizeof(a[0]))
+
+int main()
+{
+	const char str[] = "Hello, world!";
+	const char* same_str = "Hello, world!";
+	std::cout << "Our str: " << str << std::endl;
+	std::cout << "Sizeof: " << sizeof(str) << std::endl;
```

Comment:
Forgot sizeof(same_str) and sizeof(same_str1). Please add. Explain why results are different respectively from sizeof(str), sizeof(str1).

Reply:
Added